### ISSUE TITLE
In "Entity Details"-panel the Component-Controlgroup does not receive keyboard focus

### ISSUE THREAD

**Issue by Rhast0r** [EvergineTeam/Feedback#146]
When trying to Tab through controls the container control does not seem to be able to get a keyboard focus. If you were able it would be nice if you can toggle the expand/collapsed state using enter

**Comment by Trifido** [comment]
Hi @Rhast0r,
Thank you for your feedback.
We have added it to our backlog.
Regards

**Comment by jcant0n** [comment]
Thank you for taking the time to report this issue. We are closing it due to extended inactivity. If you believe this issue is still relevant, please feel free to reopen it and we will be happy to revisit it. Thank you for your understanding and for helping us improve Evergine.
